extension/src/shared/ui-components/MyData/WebDataTab.tsx

```diff
@@ -0,0 +1,206 @@
+import { Input, inputClasses } from "@mui/base/Input";
+import AddIcon from "@mui/icons-material/Add";
+import { Button, Typography, styled } from "@mui/material";
+import FormControl from "@mui/material/FormControl";
+import { ChangeEvent, Fragment, useCallback, useState } from "react";
+
+import { AdditionalData } from "../../../../../plateau-api-migrator/src/types/view2/core";
+import { getExtension } from "../../utils/file";
+import { Label } from "../Label";
+
+import { StyledButton } from "./StyledButton";
+import { UserDataItem } from "./types";
+import { getAdditionalData } from "./utils";
+import WebFileTypeSelect, { FileType, getSupportedType } from "./WebFileTypeSelect";
+
+type Props = {
+  onSubmit: (selectedItem: UserDataItem) => void;
+};
+const WebDataTab: React.FC<Props> = ({ onSubmit }) => {
+  const [fileType, setFileType] = useState<FileType>("auto");
+  const [dataUrl, setDataUrl] = useState("");
+  const [selectedWebItem, setSelectedWebItem] = useState<UserDataItem>();
+  const [requireLayerName, setRequireLayerName] = useState<boolean>(false);
+  const [layers, setLayers] = useState<string[]>([]);
+
+  const handleFileTypeSelect = useCallback((type: string) => {
+    setFileType(type as FileType);
+  }, []);
+
+  const handleSetUrl = useCallback((value: string) => {
+    setDataUrl(value);
+  }, []);
+
+  const setDataFormat = useCallback((type: FileType, filename: string) => {
+    if (type === "auto") {
+      let extension = getSupportedType(filename);
+      if (!extension) extension = getExtension(filename);
+      return extension;
+    }
+    return type;
+  }, []);
+
+  const needsLayerName = useCallback((url: string): boolean => {
+    const serviceTypes = ["mvt", "wms", "wmts"];
+    for (const serviceType of serviceTypes) {
+      if (url.includes(serviceType)) {
+        return true;
+      }
+    }
+    return false;
+  }, []);
+
+  const handleClick = useCallback(async () => {
+    // Catalog Item
+    const filename = dataUrl.substring(dataUrl.lastIndexOf("/") + 1);
+    const id = "id" + Math.random().toString(16).slice(2);
+    const format = setDataFormat(fileType, filename);
+
+    let additionalData: AdditionalData | undefined;
+    if (format === "csv") {
+      const csv = await fetch(dataUrl);
+      if (csv.status === 200) {
+        const content = await csv.text();
+        additionalData = getAdditionalData(content, format);
+      }
+    }
+
+    const item: UserDataItem = {
+      type: "item",
+      id: id,
+      dataID: id,
+      description: `著作権や制約に関する情報などの詳細については、このデータの提供者にお問い合わせください。${
+        format === "csv"
+          ? "<br/><br/>パフォーマンス上の問題が発生するため、6000レコード以上を含むCSVファイルをアップロードしないでください。"
```

Comment:
It seems `<br />` element is appear as string :eyes:
![Screenshot 2024-01-22 at 16 48 26](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/assets/34934510/4ecfdfc2-ea36-41d8-b0ad-937fd4ada5b4)